RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -487,11 +487,15 @@ boolean auto_pre_adventure()
 		acquireHP();
 	}
 
-	int wasted_mp = my_mp() + mp_regen() - my_maxmp();
-	if(wasted_mp > 0 && my_mp() > 400)
+	//my_mp is broken in Dark Gyffte
+	if (auto_my_path() != "Dark Gyffte")
```

Comment:
```suggestion
	if (my_class() != $class[Vampyre])
```

Reply:
we prefer to use class when possible because any errors in this are caught by validate. while strings like path cannot be caught.

Reply:
That seems reasonable to me, I did not know that about validate, Thanks!